docs/sdk2/usage/CATEGORY_SYNC.md

```diff
@@ -0,0 +1,467 @@
+which will improve the overall performance of the sync and commercetools API.
+
+Playing with this option can change the memory usage of the library. If it is not set, the default cache size is `10.000` for category sync.
+````java
+final CategorySyncOptions categorySyncOptions = 
+         CategorySyncOptionsBuilder.of(sphereClient).cacheSize(5000).build(); 
+````
+
+### Running the sync
+After all the aforementioned points in the previous section have been fulfilled, to run the sync:
+````java
+// instantiating a category sync
+final CategorySync categorySync = new CategorySync(categorySyncOptions);
+
+// execute the sync on your list of categories
+CompletionStage<CategorySyncStatistics> syncStatisticsStage = categorySync.sync(categoryDrafts);
+````
+The result of the completing the `syncStatisticsStage` in the previous code snippet contains a `CategorySyncStatistics`
+which contains all the stats of the sync process; which includes a report message, the total number of updated, created, 
+failed, processed categories and the processing time of the last sync batch in different time units and in a
+human-readable format.
+
+````java
+final CategorySyncStatistics stats = syncStatisticsStage.toCompletebleFuture().join();
+stats.getReportMessage(); 
+/*"Summary: 2000 categories were processed in total (1000 created, 995 updated, 5 failed to sync and 0 categories with a missing parent)."*/
+````
+
+__Note__ The statistics object contains the processing time of the last batch only. This is due to two reasons:
+
+ 1. The sync processing time should not take into account the time between supplying batches to the sync. 
+ 2. It is not known by the sync which batch is going to be the last one supplied.
+
+##### More examples of how to use the sync
+
+1. [Sync from another CTP project as a source](#todo).
+2. [Sync from an external source](#todo).
+
+*Make sure to read the [Important Usage Tips](IMPORTANT_USAGE_TIPS.md) for optimal performance.*
+
+### Build all update actions
+
+A utility method provided by the library to compare a Category with a new CategoryDraft and results in a list of category update actions. 
+```java
+List<CategoryUpdateAction> updateActions = CategorySyncUtils.buildActions(category, categoryDraft, categorySyncOptions);
+```
+
+Examples of its usage can be found in the tests 
+[here](#todo).
+
+
+### Build particular update action(s)
+
+Utility methods provided by the library to compare the specific fields of a Category and a new CategoryDraft, and in turn, build
+ the update action. One example is the `buildChangeNameUpdateAction` which compares names:
+````java
+Optional<CategoryUpdateAction> updateAction = buildChangeNameUpdateAction(oldCategory, categoryDraft);
+````
+More examples of those utils for different fields can be found [here](#todo).
+
+
+## Caveats   
+1. The library will sync all field types of custom fields, except `ReferenceType`. [#87](https://github.com/commercetools/commercetools-sync-java/issues/87). 
```

Comment:
This issue doesn't exist anymore.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/977/commits/f30b9ac93fa17ddceb7ebb409e9c8bee09debf11